src/app/project/template.html

```diff
@@ -14,312 +14,352 @@
 limitations under the License.
 -->
 
-<div fxLayout
-     fxLayoutAlign=" center">
-  <km-search-field (queryChange)="onSearch($event)"></km-search-field>
-  <div fxFlex></div>
-  <mat-slide-toggle *ngIf="isAdmin"
-                    class="all-projects-toggle"
-                    labelPosition="before"
-                    [ngModel]="settings.displayAllProjectsForAdmin"
-                    (toggleChange)="changeProjectVisibility()">Show All Projects
-  </mat-slide-toggle>
-  <mat-button-toggle-group class="project-view-switch"
-                           group="projectsView"
-                           (change)="changeView()">
-    <mat-button-toggle value="projectscard"
-                       [checked]="showCards">
-      <i class="km-icon-mask km-icon-projects-card"></i>
-    </mat-button-toggle>
-    <mat-button-toggle value="projectstable"
-                       [checked]="!showCards">
-      <i class="km-icon-mask km-icon-projects-table"></i>
-    </mat-button-toggle>
-  </mat-button-toggle-group>
-  <button id="km-add-project-top-btn"
-          mat-flat-button
-          type="button"
-          fxLayoutAlign="center center"
-          [disabled]="isProjectCreationRestricted()"
-          (click)="addProject()">
-    <i class="km-icon-mask km-icon-add"></i>
-    <span>Add Project</span>
-  </button>
+<div *ngIf="!isInitializing && !isProjectsExists else showProjectView"
+     fxLayout="column"
+     fxLayoutAlign="center center"
+     class="placeholder">
+  <div fxLayout="column"
+       fxLayoutAlign="center center">
+    <i class="km-icon-mask km-icon-projects-card i-40"></i>
+    <h2>Get Started</h2>
+    <div fxLayout="column"
+         fxLayoutAlign="center center"
+         class="placeholder-text">
+      <p>Dive right into your first project.</p>
+      <a href="https://docs.kubermatic.com/kubermatic/master/tutorials-howtos/project-and-cluster-management/"
+         target="_blank"
+         rel="noreferrer noopener">
+        Learn more in our documentation >
+      </a>
+    </div>
+    <br />
+    <button mat-flat-button
+            fxLayoutAlign="center center"
+            type="button"
+            [disabled]="isProjectCreationRestricted()"
+            (click)="addProject()">
+      <i class="km-icon-mask km-icon-add"></i>
+      <span>Create Project</span>
```

Comment:
```suggestion
      <span>Add Project</span>
```
Text should be same as regular "Add Project" button.